docs/generative_ai.md

```diff
@@ -11,14 +11,14 @@ PCA also supports 'Generative AI Queries' - which simply means you can ask quest
 
 ## Generative AI Insights
 
-When enabled, PCA can run one or more FM inferences against Bedrock or Anthropic APIs. The prompt used to generate the insights is configured in a [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html). The name of the parameter is `LLMPromptSummaryTemplate`.
+When enabled, PCA can run one or more FM inferences against Amazon Bedrock or Anthropic APIs. The prompt used to generate the insights is configured in a [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html). The name of the parameter is `LLMPromptSummaryTemplate`.
 
 ### Single FM Inference
 
 The default value for the prompt parameter provides one single prompt:
 
 ```
-Human: Answer all the questions below as a json object with key value pairs, the key is provided, and answer as the value, based on the transcript. Only return json. 
+Human: Answer all the questions below as a json object with key value pairs, based on the transcript. Use the text before the colon as the key. Only return json. Use gender neutral pronouns. Skip the preamble; go straight into the json.
```

Comment:
Per Anthropic docs, we should have double newlines.. `\n\nHuman:` to align with the model's training. Or here, `<br><br>Human:`

Reply:
Claude has been trained and fine-tuned using [RLHF (reinforcement learning with human feedback) methods](https://github.com/anthropics/hh-rlhf) on \n\nHuman: and \n\nAssistant: data like this, so you will need to use these prompts in the API i